html/global_attributes.json

```
@@ -1026,6 +1026,56 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "until-found_value": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/hidden",
+            "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute",
+            "description": "The <code>until-found</code> attribute value.",
```

@Elchi3 we don't have any lints for this, but are there conventions to follow on leading "The", punctuation, etc.?

---

Unfortunately, there are no lints here. "description" should have been "label", so this should rather be:

```suggestion
            "description": "<code>until-found</code> value",
```